azure_functions_worker/utils/common.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+
+def is_true_like(setting: str):
+    if setting is None:
+        return False
+
+    return setting.lower().strip() in ['1', 'true', 't', 'yes', 'y']
+
+
+def is_false_like(setting: str):
```

Comment:
You don't seem to need `is_false_like` and `is_envvar_false` yet, other than for symmetry. Consider removing unnecessary code for now.